source/extensions/filters/network/rocketmq_proxy/active_message.cc

```diff
@@ -165,32 +164,30 @@ void ActiveMessage::onQueryTopicRoute() {
         const auto filter_it = filter_metadata.find(NetworkFilterNames::get().RocketmqProxy);
         ASSERT(filter_it != filter_metadata.end());
         const auto& metadata_fields = filter_it->second.fields();
-        ASSERT(metadata_fields.contains(RocketmqConstants::get().BrokerName));
-        std::string broker_name =
-            metadata_fields.at(RocketmqConstants::get().BrokerName).string_value();
-        ASSERT(metadata_fields.contains(RocketmqConstants::get().ClusterName));
+        ASSERT(metadata_fields.contains(std::string(BrokerName)));
```

Comment:
I still see negative point about this part, instantiate std::string every time on data path is definitely a performance drop. It should be reverted back to use constant if string_view lookup doesn't work

Reply:
I change to use constatnt, please review again.

Reply:
Again this is still same, passing a `constexpr char[]` into `const std::string&` will call std::string constructor implicitly hence same performance drop. If `constexpr absl::string_view` you should keep the old constants. Those are not related to `well_known_names`.